### ISSUE TITLE
Geojson formatter error

### ISSUE THREAD

**Issue by nitrag** [mbloch/mapshaper#171]
Apparently the following string attribute causes an error in geojson export:

```
Manzanita - $80.00 ($88.00)* per night<br />

White Fir - $100.00 ($111.00)* per night<br />

Red Fir - $120.00 ($133.20)* per night<br />

Cedar - $&nbsp;120.00 ($133.20)* per night<br />

&nbsp;*(taxes included)
```

```json
"RECAREAURL": "http://www.fs.usda.gov/recarea/sierra/recarea/?recid=45564",
"FEEDESCRIP": "Manzanita - $80.00 ($88.00)* per night<br />\r\nWhite Fir - $100.00 ($111.00)* per night<br />\r\nRed Fir - $120.00 ($133.20)* per night<br />\r\nCedar - "
$ "nbsp;120.00 ($133.20)* per night<br />\r\n&nbsp;*(taxes included)",
"OPEN_SEASO": "May",
```
```
Error: Parse error on line 22377:
... />\r\nCedar - "			$ "nbsp;120.00 ($133
----------------------^
Expecting 'EOF', '}', ':', ',', ']', got 'undefined'
```

Would it be possible to have a function that iterates over every shape's attributes for a regex replace? I'd like to be able to strip HTML, linebreaks, and apparently these funky characters combinations  ($&)  that break json export.

**Comment by nitrag** [comment]
Not related to this is it?
https://github.com/mbloch/mapshaper/blob/master/src/geojson/geojson-export.js#L83

**Comment by mbloch** [comment]
I see why that line looks suspicious, but i'm pretty sure it's not the culprit here.

It looks like you're working with one of the Shapefiles on this page: https://data.fs.usda.gov/geodata/edw/datasets.php?dsetParent=Recreation

I confirmed that mapshaper exports invalid JSON for the FEEDESCRIP property of the one record that you pasted above. I'll look into why this is happening.

Meanwhile, a simple fix is to replace all of the &amp;nbsp; strings with regular spaces using mapshaper's `-each` command, like this:

```
mapshaper S_USA.RECREATIONOPPORTUNITIES.shp encoding=utf8 -each 'FEEDESCRIP = FEEDESCRIP.replace(/&nbsp;/g, " ")' -o out.geojson
```

**Comment by mbloch** [comment]
You're absolutely right, that line was the culprit. I wasn't aware that String.replace() treats '$&' as a special reference even when the first argument is a string literal (and not a regular expression).

I just published v0.4.8, which has a fix.

**Comment by nitrag** [comment]
Bingo! Awesome thanks! If I do ever get around to it, I'll create the feature to manipulate multiple attributes at once as mentioned previously.
